dom5.ts

```diff
@@ -38,6 +38,15 @@ export function hasAttribute(element: Node, name: string): boolean {
   return getAttributeIndex(element, name) !== -1;
 }
 
+export function hasSpaceSeparatedAttributeValue(element: Node, name: string, value: string): boolean {
```

Comment:
Although, now that I look at it, the only hasX function that isn't of the predicate function returning form is hasAttribute and we do have hasAttr.  So I'm converting this to `hasSpaceSeparatedAttrValue(name: string, value: string)`